package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@capsulajs/capsulahub-core",
+  "version": "0.0.1",
+  "description": "Core services for the CapsulaHub",
+  "main": "index.js",
+  "repository": "git@github.com:capsulajs/capsulahub-core.git",
+  "author": "MikeSquall <michael.arbib@om2.com>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "bootstrap": "lerna bootstrap --npm-client=yarn",
+    "build": "lerna run build",
+    "lint": "lerna run lint",
+    "start": "lerna run start",
+    "test": "lerna run test --concurrency=1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "homepage": "https://github.com/capsulajs/capsulahub-core",
+  "bugs": {
+    "url": "https://github.com/capsulajs/capsulahub-core/issues"
+  },
+  "dependencies": {
+    "lerna": "^3.13.4"
+  },
+  "devDependencies": {
+    "codacy-coverage": "^3.4.0",
+    "husky": "^2.2.0",
+    "prettier": "^1.17.0",
+    "pretty-quick": "^1.10.0",
+    "remark-cli": "^6.0.1",
+    "remark-lint": "^6.0.4",
+    "remark-preset-lint-recommended": "^3.0.2",
+    "rimraf": "^2.6.3",
+    "jest": "^24.8.0",
+    "ts-jest": "^24.0.2",
+    "@types/jest": "^24.0.12",
+    "tslint": "^5.16.0",
+    "tslint-config-prettier": "^1.18.0",
+    "typedoc": "^0.14.2",
+    "typescript": "^3.4.5"
```

Comment:
Don't you think that we need the majority of these packages to be installed in each package itself in order to make them work independently?

Reply:
@stephanebenayoun what do you think also?